modules/programs/zsh.nix

```
@@ -224,6 +224,16 @@ let
 
       package = mkPackageOption pkgs "zsh-syntax-highlighting" { };
 
+      highlighters = mkOption {
+        type = types.listOf types.str;
+        default = [ ];
+        example = [ "brackets" ];
+        description = ''
+          Highlighters to enable
+          See the list of highlighters: <https://github.com/zsh-users/zsh-syntax-highlighting/blob/master/docs/highlighters.md>
+        '';
+      };
+
       styles = mkOption {
         type = types.attrsOf types.str;
         default = {};
```

Let's also add an example as you noted in the issue:

```suggestion
        default = {};
        example = { comment = "fg=black,bold"; };
```